components/Card/style.module.css

```diff
@@ -11,14 +12,15 @@
   text-decoration: none;
   border: 1px solid #eaeaea;
   border-radius: 10px;
-  transition: color 0.15s ease, border-color 0.15s ease;
+  transition: color 0.5s ease, background-color 0.5s ease, border-color 0.5s ease;
```

Comment:
These much transition time makes the website feel slow. Could you try a smaller value, please?